### PR TITLE
Add pages

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: site/src/analytics-element.ts [hunk 1]
```diff
@@ -11,16 +11,22 @@ import "./analytics-active-installations";
...
   @internalProperty() private _data?: AnalyticsData;
 
+  @internalProperty() private _currentPage = "installations";
+
   @internalProperty() private _error: boolean = false;
 
   protected firstUpdated(_changedProperties: PropertyValues) {
     super.firstUpdated(_changedProperties);
+    if (window.location.hash) {
+      this._currentPage = window.location.hash.replace("#", "");
```
Comment (bramkragten) [inline site/src/analytics-element.ts:28]: I know it is WIP, but we should make sure the back button also works.

Comment (bramkragten) [reply site/src/analytics-element.ts:28]: You could then also drop the `page-changed` event

File: site/src/analytics-element.ts [hunk 2]
```diff
@@ -83,6 +94,11 @@ export class AnalyticsElement extends LitElement {
     }
   }
 
+  private _pageChanged() {
+    this._currentPage = this._currentPage =
```
Comment (bramkragten) [inline site/src/analytics-element.ts:98]: Duplicate?

File: site/src/analytics-header.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            >
+              ${page}
+            </div>`
+        )}
+      </div>`;
+  }
+
+  private _pageSelected(ev: CustomEvent) {
+    const selectedPage = (ev.currentTarget as any).page;
+    this.currentPage = selectedPage;
```
Comment (bramkragten) [inline site/src/analytics-header.ts:32]: This is already passed from the parent

File: site/src/analytics-element.ts [hunk 4]
```diff
@@ -11,18 +11,24 @@ import "./analytics-active-installations";
...
+  @internalProperty() private _currentPage = "installations";
+
   @internalProperty() private _error: boolean = false;
 
   protected firstUpdated(_changedProperties: PropertyValues) {
     super.firstUpdated(_changedProperties);
     this.getData();
+    this._pageChanged();
+    window.addEventListener("hashchange", () => this._pageChanged(), false);
+    window.addEventListener("backbutton", () => this._pageChanged(), false);
```
Comment (bramkragten) [inline site/src/analytics-element.ts:31]: What is this event?

**Review by bramkragten — CHANGES_REQUESTED** [review]

File: site/src/analytics-element.ts (hunk 4 above)
Comment (ludeeus) [reply site/src/analytics-element.ts:31]: wishful thinking

**Review by bramkragten — APPROVED** [review]

**Pull request merged**